client/network-gossip/src/bridge.rs

```diff
@@ -36,101 +37,42 @@ pub struct GossipEngine<B: BlockT> {
 struct GossipEngineInner<B: BlockT> {
 	state_machine: ConsensusGossip<B>,
 	network: Box<dyn Network<B> + Send>,
+	periodic_maintenance_interval: futures_timer::Delay,
+	network_event_stream: Compat01As03<Box<dyn Stream01<Error = (), Item = Event> + Send>>,
+	engine_id: ConsensusEngineId,
```

Comment:
Once this lands there is no need for `GossipEngineInner` anymore, thus the `engine_id` does not need to be duplicated either.